modules/opsgenie/README.md

```diff
@@ -1,338 +1,86 @@
-    description: "Infrastructure Team"
-    members:
-      - username: user1@example.com
-        role: admin
-      - username: user2@example.com
-        role: user
-
-  - name: example-team
-    description: "Example Team"
-    members:
-      - username: user3@example.com
-        role: admin
-      - username: user4@example.com
-        role: user
-```
-
-See [Opsgenie Teams](https://docs.opsgenie.com/docs/teams) for more details.
-
-<br>
-
-### `existing_users.yaml`
-
-The existing users (those that are already in Opsgenie) are described here.
-
-These users will be looked up using the data source `data "opsgenie_user"`.
-
-
-```yaml
-existing_users:
-  - username: user1@example.com
-  - username: user2@example.com
-  - username: user3@example.com
-  - username: user4@example.com
-```
-
-See [Opsgenie Users](https://docs.opsgenie.com/docs/users) for more details.
-
-<br>
-
-### `users.yaml`
-
-New users (to be created by the module) are described here.
-
-__NOTE:__ Once a user is created by the module, it's not possible to destroy it using Terraform (not supported by the Opsgenie Terraform provider).
-
-
-```yaml
-users:
-  - username: user5@example.com
-    full_name: New User
-    role: User
-    locale: "en_US"
-    timezone: "America/New_York"
-```
-
-See [Opsgenie Users](https://docs.opsgenie.com/docs/users) for more details.
-
-<br>
-
-
-### `notification_policies.yaml`
-
-Notification Policies are used to apply different operations (e.g. `delay/suppress`, `auto restart`, and `auto close`) to all team alert notifications.
-
-
-```yaml
-notification_policies:
-
-  - name: auto-close-based-on-priority
-    team_name: test
-    auto_close_action:
-      time_unit: minutes
-      time_amount: 120
-    filter:
-      type: match-all-conditions
-      conditions:
-        - field: priority
-          operation: less-than
-          expected_value: P3
-```
-
-See [Opsgenie Notification Policy](https://docs.opsgenie.com/docs/team-policies#notification-policy) for more details.
-
-<br>
-
-### `escalations.yaml`
-
-Escalations are used to escalate the alerts and incidents to a top-level Team 
-if they do not get acknowledged during the specified amount of time.
-
-Escalations are also used to notify responders according to a given order.
+**Stack Level**: Global
 
+Here's an example snippet for how to use this component. See the [detailed usage](./detailed-usage.md) documentation for the full breakdown in usage.
 
 ```yaml
-escalations:
-  - name: example-team-escalation-to-devops
-    description: "Escalate to 'devops' team if 'example-team' does not acknowledge in 10 minutes"
-    owner_team_name: example-team
-    rule:
-      condition: if-not-acked
-      notify_type: all
-      delay: 10
-      recipients:
-        - type: team
-          team_name: devops
-    repeat:
-      wait_interval: 10
-      count: 2
-      reset_recipient_states: false
-      close_alert_after_all: false
+TODO: Do we have a full example YAML snippet somewhere?
```

Comment:
@osterman @aknysh Do either of you have a good example stack snippet for this module from another project that you could share? I figured a full example from a client project that has been made generic would be best here since it seems like it's highly customizable.

Reply:
Yep! https://github.com/cloudposse/terraform-opsgenie-incident-management/tree/master/examples/config/resources

Reply:
@osterman updated 👍